# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x11/xfree86-cvs/imake_tmpl.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./config/cf/Imake.tmpl.orig	Sun Mar 10 22:44:13 2002
+++ ./config/cf/Imake.tmpl	Sun Mar 10 22:44:52 2002
@@ -1036,10 +1036,10 @@
 #define GzipCmd gzip
 #endif
 #ifndef CppCmd
-#define CppCmd /LibDirName/cpp
+#define CppCmd cpp
 #endif
 #ifndef RawCppCmd
-#define RawCppCmd CppCmd -undef
+#define RawCppCmd CppCmd
 #endif
 #ifndef CppNoLineInfoOption
 #define CppNoLineInfoOption /**/
@@ -1491,6 +1491,7 @@
 
           INCROOT = IncRoot		/* base of where to put header files */
         USRLIBDIR = UsrLibDir		/* nonshared libraries */
+        MODULEDIR = $(USRLIBDIR)/modules /* Addeed by Clifford */
            VARDIR = VarDirectory	/* usually /var */
         VARLIBDIR = VarLibDir		/* xdm runtime files */
   SYSTEMUSRLIBDIR = SystemUsrLibDir	/* system's "/usr/lib" */
